t/.pherkin.yaml

```diff
@@ -23,7 +23,7 @@ default:
              retry_timeout: 30
              driver:
                 drv_name: Weasel::Driver::Selenium2
-                wait_timeout: 3000
+                wait_timeout: 31000
```

Comment:
What's the reason to increase the wait time-out? Does it need to be increased by a factor 10? I know about the Dojo components timing out every now and then, but there were fixes for that on master recently. Maybe we don't need this increase anymore?